WcaOnRails/config/locales/en.yml

```diff
@@ -1665,72 +1665,89 @@ en:
+        content:
+          '1': "Unfortunately, the WCA has a strict policy about not allowing competitors register after the registration period has ended. Sometimes competitions have on the spot registration which allows competitors to register on the day of the event if the competition has not reached the competitor limit. You can see if a competition has on the spot registration on the competition’s website."
+      '7':
+        title: "Is it possible to go to spectate a competition without competing?"
+        #context: Answer to "Is it possible to go to spectate a competition without competing?"
+        content:
+          '1': "Most competitions allow spectators, but there might be limits or fees associated with spectating depending on the competition and venue. You can look for information about it on the competition’s website."
+      '8':
         title: "How can I have a WCA Competition in my hometown?"
         #context: Paragraph of the answer to "How can I have a WCA competition in my hometown?"
         content:
-          '1': "If you are interested in organizing a competition, it's highly recommended to attend at least one or two competitions as a competitor to learn from the experience."
-          '2': "WCA Competitions must follow the <a target='_blank' href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. The organization team doesn't have to know the regulations by heart, however, a deep knowledge of the basics of the regulations would definitely help in correctly organizing the competition."
+          '1': "The location of where competitions are held is based on where there are people who want to organize. If you are interested in organizing a competition, it's highly recommended to attend at least one or two competitions as a competitor to learn from the experience."
+          '2': "WCA Competitions must follow the <a target='_blank' href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. The organization team doesn't have to know the regulations by heart, however, a basic knowledge of the regulations would definitely help in correctly organizing the competition. It is suggested that potential organizers scope out venue options and get an idea for some dates when it would be best to hold the competition in their local area."
           '3': "After that, the organization team must contact a nearby <a target='_blank' href='https://www.worldcubeassociation.org/delegates'>WCA Delegate</a>. The supervision of a WCA Delegate is required in order to make a competition officially sanctioned by the WCA. The WCA Delegate will help guide the organization on how to host a successful competition."
           '4': "Visit the <a target='_blank' href='https://www.worldcubeassociation.org/organizer-guidelines'>WCA Competition Organizer Guidelines</a> for further information."
-      '6':
+      '9':
```

Comment:
Unfortunately, we must absolutely avoid renumbering the FAQ entries!
With that changes, the non-english version will display the previous translation of entry '9' (regarding "I am registered to a competition and I want to add additional events [...]") for the content of the "what are the WCA accounts for", which is definitely not desirable.

It may be misleading, but numbers here don't have a specific meaning, it can also be a "regular" string (for instance the current 7th entry of the FAQ is not the key `'7'`, but the key `results`.
What matters for the order displayed is the order in this file.

Could you please add the new entry where appropriate in the file, but keeping untouched the original numbers, and using a key that you find relevant for the new entry?

Reply:
Okay I understand. What would be the easiest way to make these edits?

Reply:
I think you can make them locally and add a new commit to the branch, which will be added to that pull request.
How did you make them initially? Through the github's edition interface or did you checkout the repository locally, and made the git commands manually?

In any case, the most straightforward way of fixing the entry names would probably be to edit the entry names for all new entries, and then to re-apply the initial numbering for existing entries.